regression/rt_gk_wham_translate_dim_gyrokinetic_2to3x.c

```diff
@@ -0,0 +1,1165 @@
+#include <math.h>
```

Comment:
@Maxwell-Rosen please rename this test rt_gk_wham_2xIC_3x2v_p1

translate_dim is just the name of the updater and only makes sense to developers, but the capability involves a bit more than that, and you are testing the whole thing here.